swan/scripts/01_storage.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Author: Danilo Piparo, Enric Tejedor, Pedro Maximino, Diogo Castro 2023
```

Comment:
Please add a header with this info on relevant files (i.e Docker files as well?)